config/routes.rb

```diff
@@ -3,7 +3,12 @@
   resources :cards
 
   root 'home#index'
-
+  get '/success' => 'home#success'
+  get '/error' => 'home#error'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.